.idea/deployment.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you think we should just purge the idea folder and put it in the gitignore fully?

Reply:
Nope, it's useful for importing, and the gitignore does a great job of removing personal settings